### DISCUSSION TITLE
Pull Request Interface Usability

### DISCUSSION THREAD

**Discussion by malceore** [community/community#40109] [Pull Requests] [answered]
### Select Topic Area

Product Feedback

### Body

Hello, the company I work for recently migrated to Github from a self-hosted Bitbucket instance. One of the things that struck us was that the Pull-request interface seems somewhat sub-optimal, mainly around approving and commenting on PRs.

So if I have reviewed a PR and I want to approve it I have to be on the 'Files Changed' tab, then click the 'Review Changes' drop down, then check 'Approve', and then click 'Submit Review'. Compared to in Bitbucket where from any tab I can click a single green checkbox and I'm done. This sounds minor, but this is four times the click for every single Pull-Request.
![Capture1](https://user-images.githubusercontent.com/10551604/204057983-42c4be7a-2f95-42ab-a5c4-d7220507f8bd.PNG)

As for commenting, I have to hover the left hand side until a '+' icon appears and then click that. When we used Bitbucket we could just click anywhere on the line. Then to submit the comment you have to click 'Start a Review' or 'Add single comment'. I understand the difference between them now, but it's confusing at first. I think what would make more sense is if it just said 'Comment', with an optional checkbox for 'notify reviewers'.
![capture_22](https://user-images.githubusercontent.com/10551604/204057969-c6022f4d-9999-4c53-b6d7-3a06297391c7.PNG)

Just wanted to know if there was anything on the Github roadmap to improve these aspects of pull requests? I have some crude mock-ups if you want to see what I was thinking, will attach them. Thanks!

**Comment by ossanna16** [comment, marked as answer]
Hi there @malceore and welcome to our community! Thank you for the feedback!
